include/grpc++/impl/codegen/call.h

```diff
@@ -224,7 +263,7 @@ class CallOpSendMessage {
   /// after use.
   template <class M>
   Status SendMessage(const M& message,
-                     const WriteOptions& options) GRPC_MUST_USE_RESULT;
+                     WriteOptions options) GRPC_MUST_USE_RESULT;
```

Comment:
Why change from the const reference? It'll be copied anyway into write_options_ , not referenced. Is this allowing a peephole optimization in inlining somehow?